### PR TITLE
Update agent OS support policy

### PR DESCRIPTION
**Pull request by keithduncan**
This changes the docs for each operating system (I only changed the agent v3 docs and didn’t touch the v2 ones) to steer folks towards which operating systems we support rather than the oldest operating system the agent can be installed on.

This doesn’t practically change where the agent can be installed, e.g. it probably still works on Windows XP we’re just saying that if it _happens to_ we can’t guarantee support when it goes wrong as we have to focus our efforts.

For Debian, Red Hat, and Ubuntu we can also drop the docs for using Upstart, as presently in service and supported versions of these all use systemd 🎉

### CODE REVIEW

**Comment by yob** [comment]
> For Debian, Red Hat, and Ubuntu we can also drop the docs for using Upstart, as presently in service and supported versions of these all use systemd

the best

File: pages/agent/v3/windows.md.erb
Comment (yob) [inline pages/agent/v3/windows.md.erb:3]: Is it worth mentioning the equivalent consumer windows version as well

**Review by yob — APPROVED** [review]
Looks technically accurate to me, but probably best to wait for a review from a docs person as well :+1:

**Review by plaindocs — APPROVED** [review]
Love it.

**Pull request merged**
